site/en/docs/multidevice/user-agent/index.md

```diff
@@ -2,98 +2,161 @@
+  <tr>
+    <td style="vertical-align: top;"><code>&lt;unifiedPlatform&gt;</code>
+    <td style="vertical-align: top;">
+      The intersection of <code>&lt;platform&gt;</code>,
+      <code>&lt;oscpu&gt;</code>, <code>&lt;androidVersion&gt;</code>,
+      and <code>&lt;deviceModel&gt;</code>, depending on device.
+      <p>The possible desktop values* are:
+      <ul>
+        <li><code>Windows NT 10.0; Win64; x64</code>
+        <li><code>Macintosh; Intel Mac OS X 10_15_7</code>
+        <li><code>X11; Linux x86_64</code>
+        <li><code>X11; CrOS x86_64 14541.0.0</code>
+        <li><code>Fuchsia</code>
+      </ul>
+      <p>The possible mobile values* are:
+      <ul>
+        <li><code>Linux; Android 10; K</code>
+      </ul>
+      <p><em>*These are fixed values; they will
+        not update even if a user is on an updated operating system
+        or device.</em></p>
+  <tr>
+    <td style="vertical-align: top;"><strong>Token before UA String reduction</strong>
+    <td style="vertical-align: top;"><strong>Description</strong>
+  <tr>
+    <td style="vertical-align: top;"><code>&lt;androidVersion&gt;</code>
+    <td style="vertical-align: top;">Android major version
+  <tr>
+    <td style="vertical-align: top;"><code>&lt;deviceModel&gt;</code>
+    <td style="vertical-align: top;">Android device model
+  <tr>
+    <td style="vertical-align: top;"><code>&lt;minorVersion&gt;</code>
+    <td style="vertical-align: top;">Chrome MINOR.BUILD.PATCH Refer to <a href="https://www.chromium.org/developers/version-numbers/">version numbers</a>.
+  <tr>
+    <td style="vertical-align: top;"><code>&lt;oscpu&gt;</code>
+    <td style="vertical-align: top;">Device operating system and (optionally) CPU architecture
+  <tr>
+    <td style="vertical-align: top;"><code>&lt;platform&gt;</code>
+    <td style="vertical-align: top;">Underlying device platform
+  
+</table>
+
+[Chromium.org](https://www.chromium.org/updates/ua-reduction/) has more detail.
+
+If you need data that's no longer in the UA string, the [User Agent Client Hints API](https://web.dev/migrate-to-ua-ch/) provides some, including:
+
+- `Sec-CH-UA`: browser name and major/significant version
+- `Sec-CH-UA-Mobile`: boolean value indicating a mobile device
+- `Sec-CH-UA-Platform`: operating system name
+
+This chromium.org page [maps tokens to User Agent Client Hints](https://www.chromium.org/updates/ua-reduction/#ua-token-to-ua-ch-mapping).
+
+## Examples of new UA strings in Chrome
+
+### Chrome on desktop
+
+#### Format 
+
+<pre>Mozilla/5.0 (&lt;unifiedPlatform>) AppleWebKit/537.36
+(KHTML, like Gecko)
+Chrome/&lt;majorVersion>.0.0.0 Safari/537.36</pre>
+
+#### Examples
+
+**Mac desktop**
+
+<pre>
+Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36
+(KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36
+</pre>
+
+**Windows desktop**
+
+<pre>
+Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36
+(KHTML, like Gecko) Chrome/117.0.0.0 Safari/537.36
```

Comment:
Nit: For these examples, it may be less confusing if they're all on the same major number (currently mac and windows differ)

Reply:
Hi Mike can I just change the Chrome version numbers? I got these strings from real devices which is why they differ

Reply:
Yeah - I would recommend changing them to the current version (it doesn't really matter, but it's less confusing if they're the same): So, `Chrome 119.0.0.0` for all.